test.js

```diff
@@ -0,0 +1,36 @@
+const test = require('tape')
+const noNodeConfig = require('./index')
+const co = require('co')
+const config = require('./testConfig/configData')
+const keys = ['mongo', 'redis']
+const PROCESS_ENV = process.env.NODE_ENV = 'test_env'
```

Comment:
@anuragCES in the test data below I don't see `test_env` environment, only `development` and `production` are you sure it should be `test_env`?